### PR TITLE
DNEXT-14705: Add the new changes in NIFReadEntriesExt

### PR DESCRIPTION
**Pull request by shaiknayabrasool-s**

### CODE REVIEW

**Review by raghu-madat — APPROVED** [review]

File: docs/reference/Func/NIFReadEntriesExt.md [hunk 1]
```diff
@@ -139,5 +154,6 @@ Info we want. */
 	 OSMemFree(hBuffer);
 }
 ```
+
```
Comment (vickum) [inline docs/reference/Func/NIFReadEntriesExt.md:157]: Can some "see also" be added here if possible. like API related to "NIF" . Check other NIF related API documentation for reference.

**Review by vickum — COMMENTED** [review]
looks fine to me overall.

File: docs/reference/Func/NIFReadEntriesExt.md (hunk 1 above)
Comment (shaiknayabrasool-s) [reply docs/reference/Func/NIFReadEntriesExt.md:157]: Added.

**Comment by shaiknayabrasool-s** [comment]
NIFReadEntriesExt2 new function has exposed, these changes are going to that function, so now closing the PR. and creating new PR.

**Pull request closed without merging**